common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -324,15 +324,26 @@ def setUp(self):
         """Setup."""
         super(LoginFailuresAdminTest, self).setUp()
         self.client.login(username=self.user.username, password='test')
-        user = UserFactory.create()
+        self.user2 = UserFactory.create()
```

Comment:
Shouldn't this use a non-ascii username to properly test the code?

Reply:
Good point, I added [Zażółć gęślą jaźń](https://pl.wikipedia.org/wiki/Za%C5%BC%C3%B3%C5%82%C4%87_g%C4%99%C5%9Bl%C4%85_ja%C5%BA%C5%84) :)